Login/src/main/java/com/bms/login/security/services/UserDetailsImpl.java

```diff
@@ -0,0 +1,92 @@
+package com.bms.login.security.services;
+
+import com.bms.login.models.User;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
```

Comment:
Use @Builder, @Value annotations

Reply:
I'm overriding the getters and setters from the interface. Not sure if it can be achieved using these annotations